### DISCUSSION TITLE
Display the resulting HTML in another component

### DISCUSSION THREAD

**Discussion by diegoOG09** [imzbf/md-editor-v3#288] [Q&A]
I want to display the resulting HTML in the right side (image below) in another component of mi Vue app. 
How can I do that? I don't know if I'am reading docs in a incorrect way
![image](https://github.com/imzbf/md-editor-v3/assets/63070877/a324747b-de6d-4a9d-93f4-c437013bb41c)

**Comment by imzbf** [comment]
Just replace component `MdEditor` with `MdPreview`.
